test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -879,43 +877,59 @@ func getNodePreparer(prefix string, cno *createNodesOp, clientset clientset.Inte
 	), nil
 }
 
-func createPods(namespace string, cpo *createPodsOp, clientset clientset.Interface) error {
-	strategy, err := getPodStrategy(cpo)
+// createPods creates Pods from createPodsOp.
+// Created Pods will have id in its label.
+func createPods(namespace string, cpo *createPodsOp, clientset clientset.Interface, id string) error {
+	strategy, err := getPodStrategy(cpo, id)
 	if err != nil {
 		return err
 	}
-	klog.Infof("Creating %d pods in namespace %q", cpo.Count, namespace)
+
+	klog.Infof("Creating %d pods in namespace %q with labelValue %s", cpo.Count, namespace, id)
+
 	config := testutils.NewTestPodCreatorConfig()
 	config.AddStrategy(namespace, cpo.Count, strategy)
 	podCreator := testutils.NewTestPodCreator(clientset, config)
 	return podCreator.CreatePods()
 }
 
-// waitUntilPodsScheduledInNamespace blocks until all pods in the given
-// namespace are scheduled. Times out after 10 minutes because even at the
+// waitUntilPodsScheduled blocks until all pods in the given namespace with the given
+// label are scheduled. Times out after 10 minutes because even at the
 // lowest observed QPS of ~10 pods/sec, a 5000-node test should complete.
-func waitUntilPodsScheduledInNamespace(ctx context.Context, podInformer coreinformers.PodInformer, name string, namespace string, wantCount int) error {
+// If a given namespace is empty, it will check Pods with all label.
+func waitUntilPodsScheduled(ctx context.Context, podInformer coreinformers.PodInformer, name string, namespace string, label string, wantCount int) error {
```

Comment:
From my personal opinion, the previous name is better. WDYT? Not mandatory
waitUntilPodsScheduled is for all namespaces in the list.
waitUntilPodsScheduledInNamespace for one namesapce

Reply:
I don't have a strong opinion here, so revert the change of renaming 👍